app-backend/db/src/main/scala/AnnotationProjectDao.scala

```diff
@@ -322,4 +322,29 @@ object AnnotationProjectDao
       )
     } yield stacInfo).value
 
+  def getSharedUsers(projectId: UUID) = {
+    for {
+      permissions <- AnnotationProjectDao.getPermissions(projectId)
+      ids = permissions
+        .filter(
+          _.subjectType == SubjectType.User
+        )
+        .map(_.subjectId)
+        .flatten
```

Comment:
```suggestion
        .flatMap(_.subjectId)
```

Reply:
`_.subjectId` is `ACR => Option[String]` right?

Reply:
should be